src/containers/App.js

```diff
@@ -1,13 +1,12 @@
-import React, { Component } from "react";
+import React from 'react';
 
```

Comment:
Hello Kilani, Can you please revert this to a stateful component. You can't use lifecycle hooks on a functional component.